docs/libcurl/opts/CURLOPT_SSL_OPTIONS.3

```diff
@@ -47,6 +47,12 @@ exception in the case of Windows' Untrusted Publishers blacklist which it seems
 can't be bypassed.\fP This option may have broader support to accommodate other
 SSL backends in the future.
 https://curl.haxx.se/docs/ssl-compared.html
+
+Added in 7.58.0:
+
+\fICURLSSLOPT_NO_TICKET\fP tells libcurl to disable session ticket during the
+ssl handshake. \fP This option is valid only if the CURLOPT_SSL_SESSION_FILE
+option is specified
```

Comment:
Why does this option need to be this hard connected to `SESSION_FILE` ?

Reply:
As explained above.